app/legacy_lib/mailchimp.rb

```diff
@@ -49,6 +49,17 @@ def self.signup supporter, mailchimp_list_id
 		put("/lists/#{mailchimp_list_id}/members", @options.merge(:body => body_hash.to_json))
   end
 
+  def signup_nonprofit_user (drip_email_list, nonprofit, user)
+    body_hash = @body.merge(create_nonprofit_user_subscribe_body(nonprofit, user))
+    put(drip_email_list.list_members_path, @options.merge(:body => body_hash.to_json))
+  end 
+
+  # second option
+  # def self.signup_nonprofit_user(drip_email_list, nonprofit, user)
+  #   body_hash = @body.merge(create_nonprofit_user_subscribe_body(nonprofit, user))
+  #   put(drip_email_list.list_members_path, @options.merge(:body => body_hash.to_json))
+  # end 
```

Comment:
You should be using this one because it's a "class method". Class methods are defined when you add `self.` before the method name in the definition. (There are other ways but for our purposes, that's enough detail)